checkstyle.xml

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE module PUBLIC
+  "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+  "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

```

Comment:
Could you explain, what is this link for?

Reply:
It's the validation against the proper version of DTD (document structure configuration) of chckstyle.xml file. But thanks to your question I realised that it should be other version and I have already commited changes.